server/src/core/user/services/user.service.ts

```diff
@@ -70,16 +70,14 @@ export class UserService extends TypeOrmQueryService<User> {
     );
   }
 
-  async deleteUser({
-    workspaceId: _workspaceId,
-    userId,
-  }: {
-    workspaceId: string;
-    userId: string;
-  }) {
-    const user = await this.userRepository.findBy({ id: userId });
```

Comment:
FindBy returns a list which was failing on the FE, findOneBy is the way to go. We can also use findOneByOrFail to remove the assert below but it's probably clearer to show "User not found" to the FE instead of a postgres error